tests/sdk/test_circuit.py

```diff
@@ -43,14 +44,14 @@ def test_get_content_type(openql: MagicMock, mock_file: None) -> None:
     with Circuit(platform_name="platform", program_name="program") as c:
         pass
 
-    assert c.content_type == "quantum"
+    assert c.content_type == str(AlgorithmType.QUANTUM)
```

Comment:
I think the  test here is actually correct and the fix here should be not to change the test, but to change the implementation of Circuit.content_type to return AlgorithmType.QUANTUM.value .  Similarly in the other places where enums are used.

Reply:
fixed